lib/osf-components/addon/components/subjects/browse/item/styles.scss

```diff
@@ -13,16 +16,18 @@
     padding: 0 0.5em;
 
     input {
+        margin-top: 0;
         margin-right: 0.5em;
```

Comment:
I consulted with Ask and he says make all the checkboxes grayscale. I played with the filters while screen sharing and we came up with:
```css
filter: grayscale(1) contrast(2);
```
This gives us grayscale and adequate contrast.